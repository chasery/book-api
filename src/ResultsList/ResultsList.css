.ResultsList {
  display: flex;
  flex-direction: column;
  justify-content: center;
}

.ResultsList ul {
  margin: 0;
  padding: 0;
}
