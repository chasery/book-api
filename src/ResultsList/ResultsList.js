import React from "react";
import Result from "../Result/Result";
import "./ResultsList.css";

class ResultsList extends React.Component {
  render() {
    return (
      <div className="ResultsList">
        <Result />
        <Result />
        <Result />
      </div>
    );
  }
}

export default ResultsList;
