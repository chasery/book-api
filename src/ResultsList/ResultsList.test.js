import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import ResultsList from "./ResultsList";

describe("ResultsList component", () => {
  it("renders without error", () => {
    const wrapper = shallow(<ResultsList />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
