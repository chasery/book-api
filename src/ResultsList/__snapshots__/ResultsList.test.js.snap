// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsList component renders without error 1`] = `
<div
  className="ResultsList"
>
  <Result />
  <Result />
  <Result />
</div>
`;
